scripts/netlify-build.sh

```diff
@@ -1,117 +0,0 @@
-#!/bin/sh
-
-# Make sure to set $BUILD_CONTEXT in the Netlify "Deploy Settings"
-echo "Building for Netlify. BUILD_CONTEXT: $BUILD_CONTEXT"
-
-# Fixes
-npm i -g process-nextick-args util-deprecate gauge
-
-yarn
-yarn bootstrap --core
-
-if [ "$BUILD_CONTEXT" = "DOCS" ]; then
-  pushd docs
-  yarn install
-  popd
-  yarn docs:build
-  mv docs/public netlify-build
```

Comment:
@ndelangen do we need to coordinate on the deploy because the output location is changing?